src/domains/ioweb-common/06_cdn.tf

```diff
@@ -0,0 +1,66 @@
+# core domain external rg
+data "azurerm_resource_group" "core_ext" {
+  name = format("%s-rg-external", local.product)
+}
+
+data "azurerm_dns_zone" "ioapp_it" {
+  name                = "ioapp.it"
+  resource_group_name = data.azurerm_resource_group.core_ext.name
+}
+
+# keyvault where the TLS certificate is stored
+data "azurerm_resource_group" "profile_rg" {
+  name = format("%s-profile-sec-rg", local.product)
+}
+
+data "azurerm_key_vault" "profile_kv" {
+  name                = format("%s-profile-kv", local.product)
+  resource_group_name = data.azurerm_resource_group.profile_rg.name
+}
```

Comment:
Change it as soon as https://github.com/pagopa/io-azure-devops/pull/22 is merged

Reply:
fixed in https://github.com/pagopa/io-infra/pull/661/commits/a0afe88cf40ca2215e21d36c92b64a3967135e08